gpkit/interactive/plotting.py

```diff
@@ -1,271 +1,6 @@
-            assert lright is sv
-            # this tick is too close to left nghbr.
-            # remove the left nghbr; recompute refs and distances
-            # steps: 1. update this, 2: update lefts (2x); 3: update right
-            newloc = (loc*weight + lloc*lweight)/float(weight + lweight)
-            sv[0] = newloc - left[1][0] + ldist
-            sv[1] = (newloc, ", ".join([lname, name]))
-            sv[2] += lweight
-            sv[3] = lleft
-            left[:] = [float('inf'), None, 0, None, None]
-            # be careful here, might be on a boundary
-            if lleft is not None:
-                lleft[4] = sv
-            if right:
-                right[0] = right[1][0] - newloc
-    return [t for _, t, w, _, _ in decorated if w]
-
-
-# pylint: disable=too-many-locals
-def sensitivity_plot(gp, keys=None, xmax=1, yxmax=1, ntick=25, mins=0.0):
-    """Plot percentage change in objective (cost) as variables change,
-    using sensitivity information.
-
-    Arguments
-    ---------
-    gp (GeometricProgram): a solved GP
-    keys (iterable):
-        list of variable keys to plot sensitivities for.
-        None defaults to all gp substitutions.
-    xmax (float):
-        max (and min) percentage to plot on x axis
-    yxmax (float):
-        max (and min) percentage for y axis, *as a fraction of xmax*
-    ntick (int):
-        maximum number of ticks for which tick labels don't overlap
-    mins (float): don't plot sensitivities below this value
-
-    Returns
-    -------
-    matplotlib.pyplot Figure
-    """
-    if keys is None:
-        keys = gp.solution["constants"].keys()
-    right_ticks, top_ticks = [], []
-    fig, left_ax = plt.subplots()
-    ymax = yxmax*xmax
-    left_ax.set_ylim((-ymax, ymax))
-    for k in keys:
-        s = gp.solution["sensitivities"]["constants"][k]
-        if abs(s) >= mins:
-            left_ax.plot((-xmax, xmax), (-xmax*s, xmax*s), "b", alpha=abs(s))
-            if abs(s) > yxmax:
-                top_ticks.append((ymax/s, str(k)))
-            else:
-                right_ticks.append((xmax*s, str(k)))
-    left_ax.set_xlabel("% change")
-    left_ax.set_ylabel("Approx. % change in cost")
-    left_ax.grid(True)
-
-    # now make a right-hand y axis with text labels for each sensitivity key
-    # might want to try Axes copy and ax.yaxis.tick_right()
-    right_ax = left_ax.twinx()
-    top_ax = left_ax.twiny()
-    ylim = left_ax.get_ylim()
-    right_ticks = _combine_nearby_ticks(right_ticks, lim=ylim, ntick=ntick)
-    top_ticks = _combine_nearby_ticks(top_ticks, lim=(-xmax, xmax), ntick=ntick)
-    right_ax.set_yticks([t[0] for t in right_ticks])
-    right_ax.set_yticklabels([t[1] for t in right_ticks])
-    top_ax.set_xticks([t[0] for t in top_ticks])
-    top_ax.set_xticklabels([t[1] for t in top_ticks], rotation=90)
-    right_ax.set_ylim(ylim)  # this needs to occur after set_yticks().
-    top_ax.set_xlim(left_ax.get_xlim())
-    return fig
```

Comment:
if we ever want to go back and get the sensitivity plot, do we have a good way (in git) to retrieve this code? How will we remember that it was removed by this PR? should we tag somehow?